0004-ledger-plugin-interface/0004-ledger-plugin-interface.md

```diff
@@ -187,12 +192,14 @@ For a detailed description of these properties, please see [`OutgoingTransfer`](
 #### fulfillCondition
 <code>ledgerPlugin.fulfillCondition( **transferId**:String, **fulfillment**:Buffer ) ⇒ Promise.&lt;null></code>
 
-Submit a fulfillment to a ledger. The ledger plugin or the ledger MUST automatically detect whether the fulfillment is an execution or cancellation condition fulfillment.
+Submit a fulfillment to a ledger. The ledger plugin or the ledger MUST
+automatically detect whether the fulfillment is an execution or cancellation
```

Comment:
> The ledger plugin or the ledger MUST automatically detect whether the fulfillment is an execution or cancellation condition fulfillment.

You can take that sentence out I think since we're no longer exposing cancellation conditions on the ILP layer.